packages/cozy-harvest-lib/.storybook/preview-head.html

```diff
@@ -0,0 +1,17 @@
+<style>
```

Comment:
Maybe add some context in a comment for this file, its purpose

Reply:
Ouais, parce que là je me demande pourquoi c'est nécessaire vu que tu importes les CSS de cozy-ui

Reply:
De ce que je vois les fichiers CSS n'importent pas les fonts mais les utilisent, sans ce fichier les fonts ne sont pas chargées

Reply:
Ah oui, c'est la stack qui injecte les fonts, bien vu!